### PR TITLE
Configure common permission to Item preview role

### PR DESCRIPTION
**Pull request by chinnu-focaloid**
Related to https://oat-sa.atlassian.net/browse/ADF-212
Configure common permission to Item preview role and revoke previous permission for Item content creator

### CODE REVIEW

File: migrations/Version202108040627446771_taoQtiTestPreviewer.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+namespace oat\taoQtiTestPreviewer\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\taoItems\model\user\TaoItemsRoles;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\tao\scripts\tools\accessControl\SetRolesAccess;
+
+final class Version202108040627446771_taoQtiTestPreviewer extends AbstractMigration
+{
+
```
Comment (shpran) [inline migrations/Version202108040627446771_taoQtiTestPreviewer.php:32]:
```suggestion
```

**Review by shpran — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by gabrielfs7 — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Comment by Natallia-Radkevich** [comment]
QA verified on  http://test-chinnufrancis.playground.kitchen.it.taocloud.org:48831/tao/Main/login

**Pull request merged**